app/src/main/java/com/quran/labs/androidquran/di/module/activity/PagerActivityModule.kt

```diff
@@ -33,7 +33,7 @@ object PagerActivityModule {
   @Provides
   @ActivityScope
   fun provideImageWidth(@ActivityContext context: Context, screenInfo: QuranScreenInfo): String {
-    return if (QuranUtils.isDualPages(context, screenInfo)) {
+    return if (QuranUtils.isDualPages(context, screenInfo, false)) {
```

Comment:
I have no way of injecting this value here so ignoring it for now. @ahmedre Could you elaborate a bit more on the use case here?

Reply:
discussed offline, moving forward, will consider moving this check into `QuranScreenInfo` if possible.